src/libs/actions/IOU.js

```diff
@@ -962,6 +965,20 @@ function getSendMoneyParams(report, amount, currency, comment, paymentMethodType
             },
         });
 
+        // Add optimistic personal details for recipient
+        optimisticPersonalDetailListData = {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.PERSONAL_DETAILS_LIST,
+            value: {
+                [recipient.accountID]: {
+                    accountID: recipient.accountID,
+                    avatar: UserUtils.getDefaultAvatarURL(recipient.accountID),
```

Comment:
Mayyyybe we should use `recipientAccountID` since we forced that to be a `Number` above?

Reply:
done!